source/_docs/installation/docker.markdown

```diff
@@ -39,16 +39,20 @@ $ docker run --init -d --name="home-assistant" -e "TZ=America/Los_Angeles" -v /P
 Alternatively, `docker-compose` works with any recent release of `docker-ce` on macOS. Note that (further down this page) we provide an example `docker-compose.yml` however it differs from the `docker run` example above. To make the .yml directives match, you would need to make _two_ changes: first add the equivalent `ports:` directive, then _remove_ the `network_mode: host` section. This is because `Port mapping is incompatible with network_mode: host:`. More details can be found at [Docker networking docs](https://docs.docker.com/network/). Note also the `/dev/tty*` device name used by your Arduino etc. devices will differ from the Linux example, so the compose `mount:` may require updates.
 
 ### Windows
+Docker containers are completely isolated from its Windows host system. So when you delete a container, all the changes you made to that container are also removed. If you want to have configuration files or other assets remain persistent, try mounting Windows folders on containers.
+
+Before proceeding make sure you have shared out a drive for docker to mount to.  This will allow saving of config files to persist on the local machine rather than in the docker container. (which may be destroyed when upgraded)
+https://docs.docker.com/docker-for-windows/#shared-drives
+https://docs.docker.com/docker-for-windows/troubleshoot/#verify-domain-user-has-permissions-for-shared-drives-volumes
 
 ```powershell
-$ docker run --init -d --name="home-assistant" -e "TZ=America/Los_Angeles" -v /PATH_TO_YOUR_CONFIG:/config --net=host homeassistant/home-assistant:stable
+$ docker run --init -d --name="home-assistant" -e "TZ=America/Los_Angeles" -v /PATH_TO_YOUR_CONFIG:/config -p 8123:8123 homeassistant/home-assistant:stable
```

Comment:
This is an incorrect change that will break things. Home Assistant requires to be running on the host network in order to have all features available.

While there are certainly are situations where this would be enough, we don't want users to run into surprises here.

Reply:
rgr that.  makes sense if you are using it in other ways.  reverted.